packages/nextra-theme-docs/src/styles.css

```diff
@@ -160,42 +158,19 @@ p {
         @media (prefers-contrast: more) {
           & {
             @apply border border-transparent text-gray-900;
-            &:hover {
-              @apply border-gray-900;
-            }
+            @apply hover:border-gray-900;
           }
           .dark & {
             @apply border border-transparent text-gray-50;
-            &:hover {
-              @apply border-gray-50;
-            }
+            @apply hover:border-gray-50;
           }
         }
       }
       ul {
-        @apply relative;
-        padding-left: 0.75rem;
-        margin-left: 0.75rem;
-        background-image: linear-gradient(
-          to right,
-          #e5e7eb 1px,
-          transparent 1px
-        );
-        .dark & {
-          background-image: linear-gradient(
-            to right,
-            rgba(224, 243, 255, 0.1) 1px,
-            transparent 1px
-          );
-        }
-        background-size: 1px calc(100% - 12px);
-        background-position: 0 6px;
-        background-repeat: no-repeat;
+        @apply relative dark:border-neutral-800;
+        @apply ltr:md:border-l ltr:pl-3 ltr:ml-1 rtl:md:border-r rtl:pr-3 rtl:mr-1;
```

Comment:
it's an overkill to setup a border with a width 1px using a `background-image` it's don't change more when you use simple border color
<img width="321" alt="image" src="https://user-images.githubusercontent.com/7361780/184454114-11b908e9-7544-4845-be3b-f218523ff12d.png">

Reply:
Weird the border here isn't shown in the deployment

Reply:
Nice catch! you are right, just fixed it.
This issue was after this fix 

https://github.com/shuding/nextra/pull/691/commits/1412161287510a77982b345a3d33b87064176344

Seems like we still need to update `<html />`.dir attribute for `ltr:/rtl:` modifiers declared in `styles.css`

https://github.com/shuding/nextra/blob/188209b982cd870876c4ab75d38465b638df18cd/packages/nextra-theme-docs/src/index.tsx#L149-L157